lib/auth/auth.go

```diff
@@ -2318,10 +2340,20 @@ func (a *Server) NewWebSession(req types.NewWebSessionRequest) (types.WebSession
 	if err != nil {
 		return nil, trace.Wrap(err)
 	}
-	checker, err := services.FetchRoles(req.Roles, a.Access, req.Traits)
+	roleSet, err := services.FetchRoles(req.Roles, a.Access, req.Traits)
 	if err != nil {
 		return nil, trace.Wrap(err)
 	}
+	clusterName, err := a.GetDomainName()
```

Comment:
```suggestion
	clusterName, err := a.GetClusterName()
```

Did you mean this instead?

Reply:
Yeah the `GetDomainName` stuff is a bit weird but it’s the same thing, just unwraps the extra `GetClusterName()` call. I forget where I saw it first but it seemed convenient at the time, I’m realizing now that it’s just extra confusing and doesn’t actually save any lines. I'll get rid of the uses I added here.
https://github.com/gravitational/teleport/blob/59fa9f0dae0f2be60234e422fb61eb33a2096419/lib/auth/auth.go#L615-L623